### DISCUSSION TITLE
Design Guide Review Club 🔎

### DISCUSSION THREAD

**Discussion by pavlenex** [BitcoinDesign/Guide#154] [Ideas]
Stale pull requests and one of the main reasons people quit contributing to an open-source project. Once you invest a lot of work into something and don't get :eyes: on it, it can be demotivating. People in general prefer creating to reviewing and that's fine. However that leads to pull request being stuck (bottleneck).

I'd like to start a discussion about expediting the review process. Reviewing a pull request in the BItcoin Design Guide can be done by anyone. Anyone can take a look at the content, provide feedback, fix typos and tell how certain page feels, even a newcomer can provide valuable insight on certain parts of the design guide. Reviews themselves can help people learn not only about Bitcoin but also about open-source processes and can help them learn about our structure and needs.

Recently, I've started being focused on reviews and created a page which can help people learn how to review https://bitcoin.design/guide/contribute/review/ 

I want to suggest that we start some sort of Review Club or a team, that can first of all teach newcomers how to review  and then meet once in a while to discuss pull requests and of course - review them from time to time.

Are there any community members interested in "joining" the efforts? Any ideas on how to approach this? The goal of the review club is to involve more people, have low-entry barriers and get new contributors comfortable with the FOSS processes.

**Comment by cbrennanpoole** [comment]
It's the most welcoming thought I've come to find since landing on GitHub some two years back. Not that the community isn't welcoming. Rather, intimidating. Thanks for the consideration. And thanks for the genuine sincerity.

**Comment by GBKS** [comment]
Love it. What's the best way to do this? The [Bitcoin Core PR review club](https://bitcoincore.reviews) meets one a week on IRC, for example. Are there any other examples?

**Reply by pavlenex** [reply]
I was thinking bi-weekly meetings on Jitsi and a channel on Slack that can be active every day (or maybe we can use existing #bitcoin-design)  

In the beginning, if there's no person interested in leading,  I'll be happy to organize these meetings & coach community members interested into getting involved into reviewing. I believe first few meetings will be more generic and orientated towards education and the process.

Later, if there's a passionate group of people willing to coordinate this, more than happy to hand the coordination over to them,
